akka-http-core/src/main/scala/akka/http/impl/model/parser/IpAddressParsing.scala

```diff
@@ -24,10 +24,10 @@ private[parser] trait IpAddressParsing { this: Parser ⇒
   def `ip-v6-address`: Rule1[Array[Byte]] = {
     import CharUtils.{ hexValue ⇒ hv }
     var a: Array[Byte] = null
-    def zero(ix: Int) = rule { run(a(ix) = 0.toByte) }
+    def zero(ix: Int) = rule { run(a(ix)= 0.toByte) }
```

Comment:
Strange... but not too bad I guess.

Reply:
hm indeed a bit weird... looks like bug? or weird setting?

Reply:
Yeah, looks like a bug. But there are more...

Reply:
Aah, read the diff the wrong way round for the others... So, this seems to be the only problem I can see so far.